packages/core/src/components/BubbleList/index.vue

```
@@ -51,91 +59,80 @@ const lastScrollTop = ref(0);
+  if (!container) return;
+  setContainerScrollTop(0);
+  autoScroll();
 }
 // 父组件触发滚动到指定气泡框
 function scrollToBubble(index: number) {
   const container = scrollContainer.value;
   if (!container) return;
 
   const bubbles = container.querySelectorAll('.el-bubble');
+
   if (index >= bubbles.length) return;
```

ESLint

[error] Expect newline after if

(antfu/if-newline)